packages/create-sitecore-jss/src/common/utils/helpers.ts

```diff
@@ -4,6 +4,18 @@ import path, { sep } from 'path';
 import { InitializerFactory } from '../../InitializerFactory';
 import { JsonObjectType } from '../processes/transform';
 
+// matched for proxy templates
+export const proxyAppMatcher = /node-.+-proxy/g;
```

Comment:
This pattern matches "node-headless-ssr-proxy" as well, while for headless-ssr there is a different journey